src/database.js

```diff
@@ -156,18 +158,23 @@ function setUpModels(db) {
 
     models.Pod.projectCharts = function (projectName, callback) {
       var query =
-        "SELECT UNIX_TIMESTAMP(date) AS timestamp,\
+        "SELECT timestamp, nodes, users, active_users_halfyear, active_users_monthly, local_posts, local_comments,\
+        users / nodes AS users_per_node,\
+        active_users_monthly / users AS active_users_ratio,\
+        local_posts / users AS posts_per_user,\
+        local_comments / users AS comments_per_user \
```

Comment:
I added the NULLIF function but the app wasn't crashing before the fix even with an entry with all stats to 0. Can't hurt so I added it anyway.